ui/src/app/views/profile-page/component.tsx

```diff
@@ -58,6 +62,23 @@ const validators = {
   areaOfResearch: required,
 };
 
+function isTwoFactorAuthEnabled(profile: Profile): boolean {
```

Comment:
Can we reuse these functions in the homepage when we check these things for the registration dashboard?   see here and below: https://github.com/all-of-us/workbench/blob/master/ui/src/app/views/homepage/component.tsx#L191

Reply:
👍